mojaloop/common/templates/backends/_reporting-events-db.tpl

```diff
@@ -44,16 +44,23 @@ Get reporting-events-db database.
   {{- default "nodatabase" (default .Values.global.reportingEventsDB.database .Values.reportingEventsDB.database) -}}
 {{- end -}}
 
+{{/*
+Get reportingEventsDB password.
+*/}}
+{{- define "common.backends.reportingEventsDB.password" -}}
+  {{- default "" (default .Values.global.reportingEventsDB.password .Values.reportingEventsDB.password) -}}
+{{- end -}}
+
 {{/*
 Get fully qualified reporting-events-db secret.name
 */}}
 {{- define "common.backends.reportingEventsDB.secret.name" -}}
-  {{- default (include "common.backends.reportingEventsDB.fullname" .) (default .Values.global.reportingEventsDB.secret.name .Values.reportingEventsDB.secret.name) -}}
+  {{- default "" (default .Values.global.reportingEventsDB.secret.name .Values.reportingEventsDB.secret.name) -}}
```

Comment:
Should this not return undefined?

I.e. I assume you tested the if statement in the `deployment.yaml` to use the password text instead of the secret?